### PR TITLE
dsg-2377: employer dashboard configuration object

### PR DESCRIPTION
**Pull request by derekdiep42**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

employer dashboard config object

Has only 1 entry as an example, later entries will be completed in Sergio's story. routes in tab will be completed in my current story. This is just the overall outline to get us both started.

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

**Review by thechechogomez — APPROVED** [review]
LGTM. Should we create a service to get the dashboard configuration or the tabs for a specific category?

**Comment by derekdiep42** [comment]
> LGTM. Should we create a service to get the dashboard configuration or the tabs for a specific category?

I don't think we are creating a service to get dashboard configurations. To my knowledge, in that grooming call we might not have API driven configurations.

**Comment by asultan001** [comment]
> > LGTM. Should we create a service to get the dashboard configuration or the tabs for a specific category?
> 
> I don't think we are creating a service to get dashboard configurations. To my knowledge, in that grooming call we might not have API driven configurations.

We likely won't have an API for this configuration but encapsulating the configuration behind a service may not be a bad idea so that access to it is consistent throughout the portal.

File: libs/employer/feature/dashboard/src/lib/services/dashboard.service.ts [hunk 1]
```diff
@@ -1,72 +1,19 @@
...
 @Injectable({
   providedIn: 'root',
 })
 export class DashboardService {
-  private readonly _transactionDefinitionFormModelMap: Map<string, FormConfigurationModel> = new Map();
-  private readonly _pagination = new PagingRequestModel({
-    sortBy: 'lastUpdatedTimestamp',
-    sortOrder: 'DESC',
-  });
+  private readonly dashboardConfiguration: IDashboardConfiguration = DashboardConfiguration;
```
Comment (thechechogomez) [inline libs/employer/feature/dashboard/src/lib/services/dashboard.service.ts:8]: nit: underscore on the private property

**Review by thechechogomez — APPROVED** [review]

File: libs/employer/feature/dashboard/src/lib/models/dashboard-configuration.model.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { INuverialTab } from '@dsg/shared/ui/nuverial';
+
+export interface IDashboardCategories {
+  description: string;
+  icon: string;
+  name: string;
+  route: string;
+  tabs: INuverialTab[]; // this does not contain route, routes will be constructed in https://dol-ewf.atlassian.net/browse/DSG-2335
```
Comment (davidyoung77) [inline libs/employer/feature/dashboard/src/lib/models/dashboard-configuration.model.ts:8]: Lets go with the name subCategories, like Abe suggests

**Review by davidyoung77 — APPROVED** [review]
approved, pending comments

**Pull request merged**